packages/integration-sdk-cli/src/__tests__/cli-run-failure.test.ts

```diff
@@ -72,37 +51,3 @@ test('aborts synchronization job if an error occurs', async () => {
     status: SynchronizationJobStatus.ABORTED,
   });
 });
-
-test('does not log errors that have been previously logged', async () => {
```

Comment:
This test wasn't very useful imo, and it required copious amounts of mocking to make it possible. Jest had breaking changes to some of its mocking functionality. I felt that it would be better to actually remove the test entirely instead of revamping the mocking.